src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -271,6 +279,23 @@ export default class FilterOptionsComponent extends Component {
     if (this.config.storeOnChange) {
       this.apply(this.config.isDynamic);
     }
+
+    if (!this.config.isDynamic) {
+      this.core.globalStorage.on('update', this.name, (data) => {
+        let previousOptions;
+        try {
+          previousOptions = JSON.parse(data);
+        } catch (e) {
+          console.warn(`Filter option ${data} could not be parsed`);
+        }
+        this.config.options = this.config.setSelectedOptions(
+          this.config.initialOptions,
+          previousOptions
```

Comment:
Should this stuff happen if a JSON.parse error was caught?

Reply:
Yes! Changed tytyty